marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.module.scss

```diff
@@ -13,12 +13,27 @@
 .cardStyle:hover {
     border: 1px solid #67749c;
 }
-
 .mapNameStyle {
     font-size: 20px;
     padding-bottom: 2px;
 }
-
+.addNewCardDisabled {
+    width: 300px;
+    margin-top: 9px;
+    border: 1px solid #d7d6d6;
+    align-items: center;
+    text-align: center;
+    align-content: center;
+    cursor: not-allowed !important;
+}
+.plusIconDisabled{
+    font-size: 80px;
+    margin-top: 16px;
+    color: #67749c;
+    opacity: 0.45;
+    margin-bottom: 0px;
+    cursor: pointer;
```

Comment:
Can we also disable the cursor on the disabled plus icon so it is consistent? Not sure if that was already discussed with UX. I see the cursor is also still enabled there for the Load tile as well, but I think it would make sense to have it disabled all-around.

Reply:
Good finding... I had discussion with UX team and we will be keeping banned pointer consistently. I have updated accordingly